tests/integration/cli/test_worklock_cli_functionality.py

```diff
@@ -234,7 +234,7 @@ def test_enable_claiming(click_runner,
 
     gas_limit_1 = 200000
     gas_limit_2 = 300000
-    user_input = YES + YES + str(gas_limit_1) + '\n' + NO + str(gas_limit_2) + '\n' + YES
+    user_input = YES_ENTER + YES_ENTER + str(gas_limit_1) + '\n' + NO_ENTER + str(gas_limit_2) + '\n' + YES_ENTER
```

Comment:
I'm wondering if perhaps it should be clearer something like:
`"\n".join(YES, str(gas_limit_1), NO, str(gas_limit_2), YES)`

What do you think?

Reply:
Yes, that would look much better